nomad/acl_endpoint.go

```diff
@@ -2777,3 +2783,152 @@ func (a *ACL) OIDCCompleteAuth(
 	reply.ACLToken = tokenUpsertReply.Tokens[0]
 	return nil
 }
+
+// Login RPC performs non-interactive auth using a given AuthMethod. This method
+// can not be used for OIDC login flow.
+func (a *ACL) Login(args *structs.ACLLoginRequest, reply *structs.ACLLoginResponse) error {
+
+	// The login flow can only be used when the Nomad cluster has ACL enabled.
+	if !a.srv.config.ACLEnabled {
+		return aclDisabled
+	}
+
+	// Perform the initial forwarding within the region. This ensures we
+	// respect stale queries.
+	if done, err := a.srv.forward(structs.ACLLoginRPCMethod, args, args, reply); done {
+		return err
+	}
+
+	// Measure the login endpoint performance.
+	defer metrics.MeasureSince([]string{"nomad", "acl", "login"}, time.Now())
+
+	// This endpoint can only be used once all servers in all federated regions
+	// have been upgraded to 1.5.2 or greater, since JWT Auth method was
```

Comment:
Should this read 1.5.3? Or should we just remove the specific version here and let that live in the `minACLJWTAuthMethodVersion` constant so we don't need to update it if we have to ship a security update or whatever before this goes out?